lib-android-robolectric/build.gradle

```diff
@@ -0,0 +1,50 @@
+apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'me.tatarka.retrolambda'
+
+project.archivesBaseName = 'android-robolectric'
+
+android {
+  compileSdkVersion configCompileSdkVersion
+  buildToolsVersion configBuildToolsVersion
+
+  defaultConfig {
+    minSdkVersion 9
+    targetSdkVersion configTargetSdkVersion
+    versionCode 1
+    versionName project.version
+  }
+
+  buildTypes {
+    release {
+      minifyEnabled false
+    }
+  }
+
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_1_8
+    targetCompatibility JavaVersion.VERSION_1_8
+  }
+
+  lintOptions {
+    disable 'InvalidPackage'
```

Comment:
I'm not sure but this can be a pure java project and it would not be needed anymore.